pkg/sku/cloud_sku_handler.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package sku
+
+type CloudSKUHandler interface {
+	GetSupportedSKUs() []string
+	GetGPUConfigMap() map[string]GPUConfig
+}
+
+type GPUConfig struct {
+	SKU      string
+	GPUCount float64
```

Comment:
Why would GPUCount be float64 type?

Reply:
There are some Azure GPU skus with fractional GPU count https://learn.microsoft.com/en-us/azure/virtual-machines/ngads-v-620-series

Reply:
changed to int as we decided partial gpu skus won't be supported